src/components/AssetInfo/TokenSideBarContent.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import css from "./TokenSideBar.scss";
+import TokenSideBarHolder from "./TokenSideBarHolder";
+import TokenSideBarBeneficiary from "./TokenSideBarBeneficiary";
+import TokenSideBarNoMatch from "./TokenSideBarNoMatch";
+import { transferTokenOwnership, transactionMinedReceipt } from "../../services/token";
+
+interface TokenSideBarContentProps {
+  adminAddress: string;
+  beneficiaryAddress: string;
+  holderAddress: string;
+  approvedBeneficiaryAddress: string;
+  registryAddress?: string;
+}
+
+const TokenSideBarContent = ({
+  adminAddress,
+  beneficiaryAddress,
+  holderAddress,
+  approvedBeneficiaryAddress,
+  registryAddress
+}: TokenSideBarContentProps) => {
+  const [fieldValue, setFieldValue] = useState({ newHolder: "", approvedBeneficiary: "" });
+  const [showActionLoader, toggleActionLoader] = useState(false);
```

Comment:
toggle implies that is doesnt need a param. Will be better to call this setActionLoader